lib/modules/filteReddit.js

```diff
@@ -827,6 +832,16 @@ function scanEntries(ele) {
 			const postFlair = thing.getPostFlairText();
 			const postLink = thing.getPostLink();
 
+			// Do not apply any other filters if posts belong to me
```

Comment:
How about moving this above `const postTitle`, so  the function can quit as early as possible?